src/Microsoft.AspNetCore.Http.Connections.Client/Internal/LongPollingTransport.cs

```diff
@@ -123,6 +120,9 @@ public Task StartAsync(Uri url, TransferFormat transferFormat)
                 throw;
             }
 
+            _transport.Output.Complete();
```

Comment:
Why does this matter?

Reply:
Completing the pipes was causing the poll loop to throw, how?

Reply:
> how?

It wasn't throwing, I mistook that. It was simply completing "gracefully" as though the server terminated the poll.